docker-compose.circle.yml

```diff
@@ -94,31 +94,31 @@ centos7build:
 ## Package testing nodes
 #
 wheezytest:
-  image: quay.io/dennybaa/droneunit:wheezy-sshd
+  image: stackstorm/packagingtest:wheezy
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 jessietest:
-  image: quay.io/dennybaa/droneunit:jessie-sshd
+  image: stackstorm/packagingtest:jessie
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 trustytest:
-  image: quay.io/dennybaa/droneunit:trusty-upstart
+  image: stackstorm/packagingtest:trusty
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 centos6test:
-  image: quay.io/dennybaa/droneunit:centos6-sshd
+  image: stackstorm/packagingtest:centos6
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 centos7test:
-  image: quay.io/dennybaa/droneunit:centos7-systemd
+  image: stackstorm/packagingtest:centos7
```

Comment:
> Am I missing something?

Yes, every distro could be changed to use different `init` system. For example, for `jessie-sshd` the default init system should be `systemd`, while we ship `sysvinit`, as I remember.

Since we deal with different init systems, It's better to say explicitly what system is used:

- Less assumptions
- Not everyone knows/remember which init system is default for this or that distro

See: https://hub.docker.com/search/?isAutomated=0&isOfficial=0&page=1&pullCount=0&q=systemd&starCount=0